packages/client/components/MeetingCard.tsx

```diff
@@ -185,45 +232,73 @@ const MeetingCard = (props: Props) => {
   }
   const {id: teamId, name: teamName} = team
 
+  const isRecurring = !!(meetingSeries && !meetingSeries.cancelledAt)
+  const meetingLink = isRecurring
+    ? `/meeting-series/${encodeURIComponent(meetingSeries.id)}`
+    : `/meet/${meetingId}`
+
   return (
     <CardWrapper
       ref={ref}
       maybeTabletPlus={maybeTabletPlus}
       status={status}
       onTransitionEnd={onTransitionEnd}
     >
-      <MeetingImgWrapper>
-        <MeetingImgBackground meetingType={meetingType} />
-        <MeetingTypeLabel>{MEETING_TYPE_LABEL[meetingType]}</MeetingTypeLabel>
-        <Link to={`/meet/${meetingId}`}>
-          <MeetingImg src={ILLUSTRATIONS[meetingType]} alt='' />
-        </Link>
-      </MeetingImgWrapper>
-      <MeetingInfo>
-        <TopLine>
-          <Link to={`/meet/${meetingId}`}>
-            <Name>{name}</Name>
-          </Link>
-          <Options ref={originRef} onClick={togglePortal}>
-            <IconLabel ref={tooltipRef} icon='more_vert' />
-          </Options>
-        </TopLine>
-        <Link to={`/meet/${meetingId}`}>
-          <Meta>
-            {teamName} • {meetingPhaseLabel}
-          </Meta>
-        </Link>
-        <AvatarList users={connectedUsers} size={28} />
-      </MeetingInfo>
-      {menuPortal(
-        <MeetingCardOptionsMenuRoot
-          meetingId={meetingId}
-          teamId={teamId}
-          menuProps={menuProps}
-          popTooltip={popTooltip}
-        />
-      )}
-      {tooltipPortal('Copied!')}
+      <div style={{position: 'relative'}}>
```

Comment:
+1 styled component? once we have tailwind css in place I do hope it's gonna get easier 😓

Reply:
I was hesitant to pull this into its own styled component because this div is here strictly to get the `position`s to play nice with the `StackedCard`s + `InnerCard`, i.e. this has no semantic meaning in the context of the rest of the component.